CONTRIBUTING.md

```diff
@@ -11,18 +11,23 @@ The following are required to build the devfile index container image containing
 - Docker 17.06 or later
 - Git
 
-## Stacks
+## Instructions
 
-### Contributing
+1. Open an issue in the [devfile/api](https://github.com/devfile/api) repo to track adding a new stack or sample
+2. Avoid using container images (check for references in the Devfile and Dockerfile) from registries (like DockerHub) that impose rate limiting.
```

Comment:
Can we provide more guidance, e.g. using a mirror on quay.io, to work around the problem